package.json

```diff
@@ -232,7 +232,7 @@
       "./assets/**"
     ]
   },
-  "devEngines": {
+  "engines": {
```

Comment:
This was required to get node to work on my PC (version 22.16). I can revert if it breaks things for you / it's unwanted.